### PR TITLE
Make email notification more specific and less prescriptive

### PR DESCRIPTION
**Pull request by brittag**
This change is a followup to https://github.com/18F/cg-buildpack-notify/pull/10 that updates the notification email in the following ways:

* Integrate links into the body of the email (while retaining plain text formatting).
* Be much more specific about the nature of the updates, to prevent unnecessary concern.
* Don't ask people to update immediately. Allow people to determine their own schedule. They probably already have their own required timelines for flaw remediation.
* Don't duplicate link to docs.

### CODE REVIEW

**Comment by brittag** [comment]
Related: [doc with draft](https://docs.google.com/document/d/1arfMpB2z8L8c1ROGSEomqOYO-ziJrBQno5mcQi1z_aw/edit).

**Comment by jcscottiii** [comment]
I added some debug statements to the test so it's easier to see the diffs for the tests
image
More detail here in commit msg: https://github.com/18F/cg-buildpack-notify/pull/11/commits/7e3c11fe9b7fe0f435eaff176936e6968287a652

**Comment by pburkholder** [comment]
I couldn't stomach `The following application have` so I completely changed the logic for how we handle plural apps.

For content, I changed:

> cloud.gov frequently updates the platform underneath your applications,

to

> cloud.gov frequently updates the programming language buildpacks available to our customers

because I thought that "changing the platform underneath" introduced an element not related to the buildpack/restaging question.

I also got to learn some templating 🎓 !

**Comment by pburkholder** [comment]
@jcscottiii Can you review please? Thanks

File: templates/mail/notify.txt [hunk 1]
```diff
@@ -1,20 +1,24 @@
-We recently made new security improvements to cloud.gov.
+Hi cloud.gov user,
 
-To take advantage of them, please restage your {{.AppNoun}} immediately [1].
+cloud.gov frequently updates the programming language buildpacks available to our customers, and we recently updated 
+{{- if .AppPlural}} the buildpacks that your applications use. {{end -}} 
+{{- if not .AppPlural}} the buildpack that your application uses. {{end -}}
```
Comment (jcscottiii) [inline templates/mail/notify.txt]: You can use `else`

File: templates/mail/notify.txt [hunk 2]
```diff
@@ -1,20 +1,24 @@
...
+{{- if not .AppPlural}} the buildpack that your application uses. {{end -}} 
+Buildpack updates include programming language updates and often include security fixes.
 
-To do so, open the command line and enter the following for each app: "cf restage YOURAPPNAME"
+To take advantage of buildpack updates, you should restage your application{{if .AppPlural}}s{{end}}. Documentation for this process: https://cloud.gov/docs/getting-started/app-maintenance/
+
+You can restage apps by opening the command line and entering the following command: cf restage YOURAPPNAME
+{{if .AppPlural}}
+The following applications have updated buildpacks available:
+{{end -}}
```
Comment (jcscottiii) [inline templates/mail/notify.txt]: Likewise, you can use else here.

**Pull request merged**